robot/Cumulus/resources/locators_51.py

```diff
@@ -325,6 +325,10 @@
         'remove_lookup':'//div[@class="modal-body scrollable slds-modal__content slds-p-around--medium"]//*[contains(@data-qa-locator,"button Remove selected option Opportunity: Record Type")]',
         'click-dropkown-value':"//div[@class='modal-body scrollable slds-modal__content slds-p-around--medium']//*[contains(@class,'slds-combobox')]//span[text()='{}'']",
         'modal_lookup_button':'//div[contains(@class,"modal-body")]//*[contains(@data-qa-locator,"{}")]//child::button',
+        'temp_builder_labels':'//lightning-input[@class="slds-truncate slds-form-element"]//label',
+        'ge_form_labels':'//label[@class="slds-form-element__label" or @class="slds-form-element__label slds-no-flex"]',
```

Comment:
can u instead use //label[contains(@class,"slds-form-element__label")] instead?

Reply:
class should be all small, github is making 'c' capital

Reply:
//label[contains(@Class,"slds-form-element__label") returns other text/label values I don't want returned in the list.